src/win/process.c

```diff
@@ -250,6 +250,30 @@ static WCHAR* search_path_join_test(const WCHAR* dir,
   return NULL;
 }
 
+/*
+ * Gets a string which contains the full path to cmd.exe
+ * caller responsible for freeing memory on success
```

Comment:
This should either be an internal-only API, or the user shouldn't be responsible for allocating the buffer.
The reason is that the user `free()` function may not work with the allocator used for `uv__alloc()`.
cc @bnoordhuis - you agree?